resources/migrations/0016_auto_20151028_1653.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import autoslug.fields
+import resources.models.utils
+
+
+class Migration(migrations.Migration):
```

Comment:
Yeah, because the slug fields need to be unique, and it cannot be set in the first migration.